sensor/common/sensor/sensor.go

```diff
@@ -179,6 +179,7 @@ func (s *Sensor) Start() {
 			Compression:   false, // kernel objects are compressed
 		}
 		customRoutes = append(customRoutes, koCacheRoute)
+		s.AddNotifiable(WrapNotifiable(probeDownloadHandler))
```

Comment:
nit: Maybe it makes more sense to provide the component name at `WrapNotifiable` rather than expect the struct to implement a `ComponentName` function? Since the component name is only needed for logging purposes, it looks like a leaky abstraction to force a `pkg/probeupload` struct to implement that. 

```suggestion
		s.AddNotifiable(WrapNotifiable("Kernel probe server handler", probeDownloadHandler))
```

Reply:
That is a good suggestion! Let me change that.